resources/charlie/validator_keys/README.md

```diff
@@ -0,0 +1,19 @@
+## Validator Keys  
+
+Keys should be created by running (assuming you are in this directory for local path "."):
+
+`sudo casper-client keygen .`
+
+This will create `public_key_hex`, `public_key.pem`, and `secret_key.pem`. 
```

Comment:
Probably worth a line about never distributing the `secret_key.pem` file (except maybe to Clarity)?